src/model/env

```diff
@@ -1,4 +1,5 @@
 DATABASE_URL="postgresql://[user]:[password]@localhost:5432/[databasename]"
 STAGING_DATABASE_URL="postgres://[user]:[password]@[host]:5432/[databasename]"
+TESTING_DATABASE_URL="postgres://[user]:[password]@[host]:5432/[databasename]"
```

Comment:
If you call your `NODE_ENV` `test`, this should be called `TEST_DATABASE_URL`.

Reply:
Fixed, good catch!